### PR TITLE
[tests-only][full-ci]implement separate step for dropdown-menu and batch-action for copy and move

### PR DESCRIPTION
**Pull request by PrajwolAmatya**
## Description
Existing tests for copying and moving files using `dropdown-menu` implements following steps:
1. right clicks on the resource
2. clicks on `Copy` button on `app bar batch action`
3. pastes on the new location

This flow implementation does not seem proper as after right click on the resource we are not copying the file from the dropdown menu, instead, we are copying files from `app bar`.
In this PR, proper implementation for copying/moving files using `dropdown-menu` is done and added an extra step for copying/moving files using the button from `app bar`.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] e2e tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/e2e/support/objects/app-files/resource/actions.ts [hunk 1]
```diff
@@ -103,6 +103,7 @@ const onlyOfficeCanvasEditorSelector = '#id_viewer_overlay'
 const onlyOfficeCanvasCursorSelector = '#id_target_cursor'
 const collaboraCanvasEditorSelector = '.leaflet-layer'
 const textEditorTextArea = '#text-editor-input'
+const filesActionDropdown = `//div[contains(@class, 'tippy-box')]//button[contains(@class, 'oc-files-actions-%s-trigger')]`
```
Comment (saw-jan) [inline tests/e2e/support/objects/app-files/resource/actions.ts:106]: we have this selector `filesAction` in line 40. Will it work?

Comment (saw-jan) [reply tests/e2e/support/objects/app-files/resource/actions.ts:106]: ah, that's for batch action button

Comment (saw-jan) [inline tests/e2e/support/objects/app-files/resource/actions.ts:106]:
```suggestion
const filesContextMenuAction = "#oc-files-context-menu button.oc-files-actions-%s-trigger"
```

Comment (saw-jan) [reply tests/e2e/support/objects/app-files/resource/actions.ts:106]: Also please, update this too https://github.com/owncloud/web/blob/0846a0a84ff57ae29c60de4d5cb327c9185761fe/tests/e2e/support/objects/app-files/resource/actions.ts#L40
to
```js
const filesBatchAction = ".files-app-bar-actions .oc-files-actions-%s-trigger"
```

Comment (PrajwolAmatya) [reply tests/e2e/support/objects/app-files/resource/actions.ts:106]: `filesAction` has same class name and resolves into two elements.

**Review by SagarGi — COMMENTED** [review]
changes LGTM :+1:

File: tests/e2e/support/objects/app-files/resource/actions.ts [hunk 2]
```diff
@@ -103,6 +103,8 @@ const onlyOfficeCanvasEditorSelector = '#id_viewer_overlay'
 const onlyOfficeCanvasCursorSelector = '#id_target_cursor'
 const collaboraCanvasEditorSelector = '.leaflet-layer'
 const textEditorTextArea = '#text-editor-input'
+const filesContextMenuAction = '#oc-files-context-menu button.oc-files-actions-%s-trigger'
+const filescontextMenuPasteAction = '.oc-files-actions-copy-trigger'
```
Comment (saw-jan) [inline tests/e2e/support/objects/app-files/resource/actions.ts:107]: ?

File: tests/e2e/support/objects/app-files/resource/actions.ts [hunk 3]
```diff
@@ -796,7 +810,7 @@ export const renameResource = async (args: renameResourceArgs): Promise<void> =>
   }
 
   await page.locator(util.format(resourceNameSelector, resourceBase)).click({ button: 'right' })
-  await page.locator(util.format(filesAction, 'rename')).click()
+  await page.locator(util.format(filesBatchAction, 'rename')).click()
```
Comment (saw-jan) [inline tests/e2e/support/objects/app-files/resource/actions.ts:813]: I think this should be
```suggestion
  await page.locator(util.format(filesContextMenuAction, 'rename')).click()
```

File: tests/e2e/support/objects/app-files/resource/actions.ts (hunk 2 above)
Comment (PrajwolAmatya) [reply tests/e2e/support/objects/app-files/resource/actions.ts:107]: Its actually for pasting the resource using dropdown-menu. I missed to add it in previous commit and added in later commit.

Comment (saw-jan) [reply tests/e2e/support/objects/app-files/resource/actions.ts:107]: I think, `filesContextMenuAction` should work, right?

Comment (PrajwolAmatya) [reply tests/e2e/support/objects/app-files/resource/actions.ts:107]: only if we use classname `tippy-box`. id for dropdown while pasting is different

Comment (saw-jan) [reply tests/e2e/support/objects/app-files/resource/actions.ts:107]: then maybe we can use for all
```js
const filesContextMenuAction = 'div[id^="context-menu-drop"] button.oc-files-actions-%s-trigger'
```

**Review by saw-jan — APPROVED** [review]
LGTM :+1:

**Pull request merged**